tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc

```diff
@@ -2595,10 +2596,10 @@ std::vector<string> GetDiscSupportedOps() {
     "Less",
     "LessEqual",
     "Log",
+    "LogSoftmax",
```

Comment:
do we need a log_softmax ut ?

Reply:
`LogSoftmax` is not introduced by this PR actually. I just sorted the ops in alphabetic order.